File: aggregator/src/main/java/com/thalesminussi/aggregator/model/station/WeatherResponseDto.java
package com.thalesminussi.aggregator.model.station;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import java.util.Collection;

@JsonIgnoreProperties(ignoreUnknown = true)
public class WeatherResponseDto {

    private Main main;

    private Collection<Weather> weather;

    private Wind wind;

    public Main getMain() {
        return main;
    }

    public void setMain(Main main) {
        this.main = main;
    }

    public Collection<Weather> getWeather() {
        return weather;
    }

    public void setWeather(Collection<Weather> weather) {
        this.weather = weather;
    }

    public Wind getWind() {
        return wind;
    }

    public void setWind(Wind wind) {
        this.wind = wind;
    }
}

File: station/src/main/kotlin/com/thalesminussi/station/model/WeatherResponseDto.kt
package com.thalesminussi.station.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import java.io.Serializable

@JsonIgnoreProperties(ignoreUnknown = true)
class WeatherResponseDto : Serializable {

    val main: Main? = null

    val weather: Collection<Weather>? = null

    val wind: Wind? = null
}